cmd/smd/smd-api.go

```diff
@@ -37,7 +37,7 @@ import (
 	"github.com/OpenCHAMI/smd/v2/internal/hmsds"
 	"github.com/OpenCHAMI/smd/v2/pkg/rf"
 	"github.com/OpenCHAMI/smd/v2/pkg/sm"
-	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/v5"
```

Comment:
We also need to do this in:
```diff
diff --git a/cmd/smd/main.go b/cmd/smd/main.go
index c7eab06..d120d0b 100644
--- a/cmd/smd/main.go
+++ b/cmd/smd/main.go
@@ -46,7 +46,7 @@ import (
        "github.com/OpenCHAMI/smd/v2/internal/slsapi"
        "github.com/OpenCHAMI/smd/v2/pkg/rf"
        "github.com/OpenCHAMI/smd/v2/pkg/sm"
-       "github.com/go-chi/chi"
+       "github.com/go-chi/chi/v5"
        "github.com/go-chi/jwtauth/v5"
        "github.com/hashicorp/go-retryablehttp"
        "github.com/sirupsen/logrus"
diff --git a/cmd/smd/routers.go b/cmd/smd/routers.go
index 2e62650..d213fa3 100644
--- a/cmd/smd/routers.go
+++ b/cmd/smd/routers.go
@@ -30,8 +30,8 @@ import (
        "strings"
        "time"
 
-       "github.com/go-chi/chi"
-       "github.com/go-chi/chi/middleware"
+       "github.com/go-chi/chi/v5"
+       "github.com/go-chi/chi/v5/middleware"
        "github.com/go-chi/jwtauth/v5"
        "github.com/gorilla/handlers"
        "github.com/lestrrat-go/jwx/jwk"
```
With just changing it in `cmd/smd/smd-api.go`, the tests still fail.

Then, run `go mod tidy`.

Reply:
Could you also put it here?

https://github.com/davidallendj/smd/blob/ee64bf92df384cd7935b25edccf970f4dbb2cdb7/cmd/smd/main.go#L49

Reply:
Looks good.